lib/entities/scheduled-action.ts

```diff
@@ -28,12 +28,13 @@ enum ScheduledActionStatus {
   canceled = 'canceled',
 }
 
-type SchedulableEntityType = 'Entry'
+type SchedulableEntityType = 'Entry' | 'Asset' | 'Release'
 type SchedulableActionType = 'publish' | 'unpublish'
+type ScheduledActionType = 'ScheduledAction' | 'ReleaseAction'
 
 export type ScheduledActionSysProps = {
   id: string
-  type: 'ScheduledAction'
```

Comment:
This should remain ScheduledAction. ReleaseAction should be a separate shape for when a release is executed

Reply:
Oh I see. Ok, thanks for noting that